common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.java

```diff
@@ -66,6 +69,16 @@
             BROKER_HOST_APP_SIGNATURE
     );
 
+    public static final BrokerData INTUNE_NGMS_AGENT_DEBUG = new BrokerData(
+            INTUNE_NGMS_AGENT_APP_PACKAGE_NAME,
+            INTUNE_NGMS_AGENT_APP_DEBUG_SIGNATURE
+    );
```

Comment:
INTUNE_NGMS_AGENT afaik is NOT a broker....and therefore these constants should NOT belong here...